app/controllers/comments_controller.rb

```diff
@@ -1,2 +1,28 @@
 class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    @user = User.find(1)
```

Comment:
- [ ] Note that the comment should not only be created for the user with the `id: 1` instead it should be created using the id of the current user which is in the params. This current user can be obtained by setting it correctly the routes. example :point_down: 

```rb
Rails.application.routes.draw do
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show: :new, :create] do
      resources :comments, only: %i[create new update destroy]
      resources :likes, only: %i[create]
    end 
  end

  # Defines the root path route ("/")
  root "users#index"
end